### DISCUSSION TITLE
[Вопрос] Будет ли затронута тема сетевых протоколов (например HTTP)?

### DISCUSSION THREAD

**Discussion by vabka** [skbkontur/dotEducation#5] [Идеи]
Часто замечаю, что некоторым новичкам сложно понять, как устроен тот же HTTP, что мешает им самостоятельно, например, переписать код с HttpWebRequest на HttpClient.

Причём незнание не на уровне "Что делает Keep-Alive" и "Чем HTTP/2 отличается от HTTP/1", а на уровне "Что такое HTTP Header"

Мне кажется, это не связано напрямую с .NET, но было бы полезно это как-то упомянуть, так как с этим часто приходится сталкиваться в работе

**Comment by nevoroman** [comment]
Определенно да, сейчас ищем эксперта для этой темы. Когда найдем его, все с ним обсудим и будем готовы анонсировать — отпишусь здесь со всей информацией.
